lib/devcycle-ruby-server-sdk/api/client.rb

```diff
@@ -23,6 +24,7 @@ def initialize(sdkKey, dvc_options = Options.new, wait_for_init = false)
         @local_bucketing = LocalBucketing.new(@sdkKey, dvc_options, wait_for_init)
         @event_queue = EventQueue.new(@sdkKey, dvc_options.event_queue_options, @local_bucketing)
       end
+      @open_feature_provider = Provider.new(self)
```

Comment:
can this atleast be a `getOpenFeatureProvider` method like the other SDKs that only creates it when first accessed.

Reply:
That's not to the ruby language spec - this is the proper way of doing it